frontend/lib/src/util/utils.test.ts

```diff
@@ -113,6 +115,9 @@ describe("embedParamValues", () => {
     "show_footer",
     "light_theme",
     "dark_theme",
+    "hide_loading_screen",
+    "show_loading_screen_v1",
+    "show_loading_screen_v2",
```

Comment:
Added these embed options, so we can turn on the old behavior, or even remove the loading skeleton altogether.